lib/CLI/Parser.hs

```diff
@@ -11,9 +11,14 @@ module CLI.Parser
 
 import BackendName (BackendName, newBackendName)
 import CLI.Types
+  (Command(..), CopyOptions(CopyOptions), CreateOptions(CreateOptions),
+  DeleteFieldOptions(DeleteFieldOptions), DeleteOptions(DeleteOptions), Direction(..),
+  FieldInfo(FieldInfo), Filter(..), FilterField(FilterFieldByContents, FilterFieldByDate),
+  FilterOp(..), FindOptions(FindOptions), Options(Options), RenameOptions(RenameOptions),
+  SetFieldOptions(SetFieldOptions), SomeCommand(..), Sort(..), TagOptions(TagOptions),
+  ViewOptions(ViewOptions), parseFilterDate)
```

Comment:
No need for explicit import list here

Reply:
But stylish-haskell made it :>

Reply:
Are you sure? AFAIK stylish-haskell doesn't turn implicit import lists into explicit.

I think you may have accidentally clicked the "code lens" that HLS adds above the import:

![image](https://user-images.githubusercontent.com/718166/169791242-faafd253-b630-47be-9ca1-28e95da2bd7e.png)

If you click that code lens (in grey), HLS will make the import list explicit.

Can you please revert this?

Reply:
We are using specific `stylish-haskell` version (0.13.0.0 AFAIR)